benchmarking/commons/hpo_main_local.py

```diff
@@ -64,12 +64,7 @@ def parse_args(
         extra_args = extra_args.copy()
     extra_args.extend(
         [
-            dict(
-                name="benchmark",
-                type=str,
-                default="resnet_cifar10",
-                help="Benchmark to run",
-            ),
+            dict(name="benchmark", type=str, help="Benchmark to run", required=True),
```

Comment:
I want users to avoid launching the default benchmark when unspecified. It helps with those examples where a custom benchmark is added - users are notified they need to specify a new benchmark. I am also thinking that it might be a good swithc since it is better that users need tp specify a benchmark they want to use rather than defaulting to cofar10. Hapyp to discuss more.

Reply:
No, we can make this change.